spec/index.bs

```diff
@@ -3551,6 +3551,7 @@ dictionary GPUTextureCopyView {
     required GPUTexture texture;
     GPUIntegerCoordinate mipLevel = 0;
     GPUOrigin3D origin = {};
+    GPUTextureAspect aspect = "all";
```

Comment:
I'm not sure this is implementable on Metal: https://developer.apple.com/documentation/metal/mtlblitcommandencoder/1400754-copy